client/src/routes.js

```diff
@@ -7,7 +7,7 @@ import AuthenticatedRoutes from "./Components/PrivateRoutes/AuthenticatedRoutes"
 
 export default (
   <Switch>
-    <Route exact path="/login" component={LoginView} />
+    <Route exact path="/" component={LoginView} />
```

Comment:
Donovan mentioned before that for readability and that the homeView should be '/' and login '/login' (I had it switched too)